packages/site-demo/content/product/components/button/index.mdx

```diff
@@ -40,6 +40,26 @@ Use full width when you need to get the button full width.
 
 <DemoBlock vAlign="center" demo="full-width" withThemeSwitcher />
 
+### Accessibility concerns
+
+#### Button
+
+Button will use the text within the button as the accessible name, if there is no text in it (replaced by an image, an icon, etc ...) then you **should pass an `aria-label` describing what the button does.
+
+Buttons may have 3 specific accessibility use cases:
+
+- **Button as a link** if you set the `linkAs` or the `href` prop, then your button will use the `<a>` tag.
+You **should define the `alt` attribute**, with a string indicating where the link will take the user.
+- **Toggle button**, when your button as an active/inactive state (`isSelected` prop is responsible of this state). 
+If the label of the button **changes depending on the state, then it doesn't need anything more.**
+But if the label of the button **doesn't change you should add a `aria-pressed=true/false`** to specify to the user in which state the button is.
+ - **Menu button** if your button opens a dropdown. 
+ You have to set, on the button, `aria-haspopup` to `true`, and `aria-expanded` to `true` when the dropdown is open.
```

Comment:
"Menu" is a very specific ARIA pattern, should we be more general and describe the popup use case (menu, combobox, dialog, etc.) and separately describe the expanded use case (menu, disclosure, popup, etc.) ?
Also it looks like `aria-expanded` should always be paired with `aria-controls` or `aria-owns`

Reply:
I took the [`Action Menu Button`](https://w3c.github.io/aria-practices/examples/menu-button/menu-button-actions.html) as a reference, this use case is listed in the [Button spec](https://w3c.github.io/aria-practices/#button)
I'm not sure what you mean, for dialog pattern I think it should be specified in the Dialog component. Maybe we can add here some links to the different DS components docs, to list all specific patterns 

`aria-controls` can't refer to an element that is not in the DOM, not sure this is relevant in our case, but I may be wrong

Reply:
Ok just read that: 
> The aria-controls only needs to be set when the popup is visible, but it is valid and easier to program to reference an element that is not visible.
I'll add this

Reply:
> Also, when the dropdown is open, you have to add `aria-controls` on the button, referencing the id of the dropdown. 
 If your dropdown is not present in the DOM when it is closed, then you should not provide the `aria-controls` prop.

 Does it looks ok for you?

Reply:
Yep, it should be sufficient